### PR TITLE
feat: minimal types for barnard59

### PR DESCRIPTION
**Pull request by tpluscode**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/barnard59-core/package.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "private": true,
+    "dependencies": {
+        "rdf-js": "^4.0.2",
```
Comment (peterblazejewicz) [inline types/barnard59-core/package.json:4]: only this little detail, why this dependency? It's not in original package deps list:
https://github.com/zazuko/barnard59-core/blob/master/package.json

Comment (tpluscode) [reply types/barnard59-core/package.json:4]: This is a types-only package, which defines data interfaces used throughout this and related packages.

But you have a point. In actual typings I import from `@rdfjs/types` (one proxies the other). Might actually align that

Comment (tpluscode) [reply types/barnard59-core/package.json:4]: Ah, shoot, it's not in the allowed list. I reverted this for now if that's ok with you.

For the future: https://github.com/microsoft/DefinitelyTyped-tools/pull/310

**Review by peterblazejewicz — APPROVED** [review]
LGTM!

**Comment by tpluscode** [comment]
Ready to merge


Sent from Nine
________________________________

**Pull request merged**